scripts/populate_preprint_providers.py

```diff
@@ -60,7 +59,13 @@ def set_or_update_m2m_fields(provider, emails, licenses, links, social_accounts)
         provider.licenses_acceptable.add(*licenses)
 
     for link in links:
-        obj, created = PreprintProviderLink.objects.get_or_create(**link)
+        obj, created = provider.links.update_or_create(
```

Comment:
Just FYI:
If you have a `{url:'Facebook.com', description:'Facebook', linked_text:'Go to the face space!'}` in the database and someone tries to add `{url:'Facebook.com', description:'Facebook', linked_text:'FACESPACE IS THE BEST'}` it's going to replace the first one since linked_text is in defaults.